src/libostree/ostree-bootloader-zipl.c

```diff
@@ -19,10 +19,18 @@
 
 #include "ostree-sysroot-private.h"
 #include "ostree-bootloader-zipl.h"
+#include "ostree-deployment-private.h"
 #include "otutil.h"
-
+#include <systemd/sd-journal.h>
 #include <string.h>
 
+#define SECURE_EXECUTION_BOOT_IMAGE     "/boot/sd-boot"
+#define SECURE_EXECUTION_HOSTKEY_PATH   "/etc/se-hostkeys/"
+#define SECURE_EXECUTION_HOSTKEY_PREFIX "ibm-z-hostkey"
+#define SECURE_EXECUTION_LUKS_ROOT_KEY  "/etc/luks/root"
+#define SECURE_EXECUTION_LUKS_CONFIG    "/etc/crypttab"
```

Comment:
Thanks for making these explicit constants.  I think there's a lot of stuff going on here, and we'll want to include links to documentation about each of these, and carefully think about how we "tie together" these things.

That said, at a broad level I'm generally OK to merge this, since the goal is to gain more experience and we can refine it over time.